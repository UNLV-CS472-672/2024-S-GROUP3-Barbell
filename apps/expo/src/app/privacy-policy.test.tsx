import { render, screen } from '@testing-library/react-native'

import PrivacyPolicy from './privacy-policy'

test('Privacy Policy Page', () => {
  render(<PrivacyPolicy />)
  expect(screen.toJSON()).toMatchSnapshot()
})
