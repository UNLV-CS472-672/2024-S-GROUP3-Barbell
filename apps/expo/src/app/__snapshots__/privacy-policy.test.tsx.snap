// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy Policy Page 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
>
  <Text
    style={
      {
        "padding": 30,
        "textAlign": "center",
      }
    }
  >
    PrivacyPolicy!
  </Text>
</RNCSafeAreaView>
`;
